### ISSUE TITLE
Issues cannot be loaded

### ISSUE THREAD

**Issue by kkitadate** [jasperapp/jasper#209]
Issues cannot be loaded, probably because an issue author is null.
(Version: 1.0.5 for MacOS)

In DevTools(Main) console, the following messages was output.
It seems that an issue author can be null.

```
Uncaught (in promise) TypeError: Cannot read property 'login' of null
    at Function.getInvolves (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Library/GitHub/V4/GitHubV4IssueClient.js:40)
    at Function.injectV4ToV3 (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Library/GitHub/V4/GitHubV4IssueClient.js:17)
    at StreamClient.injectV4Properties (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamClient/StreamClient.js:187)
    at async StreamClient.search (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamClient/StreamClient.js:128)
    at async StreamClient.exec (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamClient/StreamClient.js:73)
    at async _StreamPolling.run (/Applications/Jasper.app/Contents/Resources/app.asar/src/Renderer/Repository/Polling/StreamPolling.js:118)
```

**Comment by kaito0228** [comment]
Please merge this PR.
If this is not merged, we will not be able to use jasper forever.

**Comment by kkitadate** [comment]
> This issue seemed to be fixed at this commit [28147c4](https://github.com/jasperapp/jasper/commit/28147c441a463eb97febd9839b2398c5e4a4328e) .

I'll close this issue.
